### ISSUE TITLE
Theme related Timeline widget rendering issue 

### ISSUE THREAD

**Issue by gcuofano** [insideout10/wordlift-plugin#491]
Even if there are enough entities to show the timeline the widget appears empty. 

Here the page where the bug was firstly reported
http://mobilitadolce.net/?page_id=513

Here the markup printed (even removing the inline `display: none` the widget won't render)
```html
<div class="wl-timeline-container" style="">
   <div class="wl-timeline" id="wl-timeline-58c92da07944c" style="display: none;"></div>
</div>
```

**Comment by ziodave** [comment]
Are the event entities bound to posts or pages? This looks related to https://github.com/insideout10/wordlift-plugin/issues/482

**Comment by cyberandy** [comment]
Events in this case are bound to a page. We would need the `global=true` option to work on both pages and posts. Yes it's related to #482

**Comment by ziodave** [comment]
It's already solved in #482 then, I am closing this as duplicate.
